src/components/sdm/SelectControl.js

```diff
@@ -43,7 +45,7 @@ const SelectControl = sources => {
       div({},[
         Select({
           isOpen, label, value, options,
-          className: ['input', ...classNames].join('.'),
+          className: ['input', ...classNames].join(' '),
```

Comment:
This is the correct way to apply multiple classNames here, and I'm hesitant to say for sure, but I think this is somehow contributed to the fix